conf.d/server.conf.tmpl

```diff
@@ -185,6 +185,23 @@ server {
     ${HTTP_API_DEPLOY_CONFIG}
   }
 
+  location /api-public {
```

Comment:
This needs to be /api to cover off both admin and public. For reference, /api/public/document and /api/document => our same back-end pod that serves out both public and protected api endpoints.

Reply:
We discussed in a chat.  To summarize: /api now belongs to new version.  /api-public points at old esm-server and we update the app code to use that.  Later we can create the same functionality in the new eagle code and pull out esm-server completely.